### PR TITLE
Cs-6 Ticket

### PR DESCRIPTION
**Pull request by PaulSoonMinChoi**

### CODE REVIEW

File: app/src/components/Preferences/PreferencesCard.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+          <Typography variant='h6' className={classes.subBody}>
+            Edit
+          </Typography>
+        </div>
+        <div
+          style={{
+            display: 'flex',
+            alignItems: 'center',
+            paddingTop: '5px',
+            gap: '5px',
```
Comment (jonathanbutler7) [inline app/src/components/Preferences/PreferencesCard.js:31]: What does `gap` do? Just curious.

**Comment by jonathanbutler7** [comment]
Looking really good! 

Could you attach a few screenshots when there is a UI change like this one? I just want to take a look at a couple things tomorrow when I'm at my computer.

Also please include a description of what this PR does in the future. A format I like to use is :

This PR:
- Does this thing
- And this other thing

# Include UI screenshots here

File: app/src/components/Preferences/PreferencesCard.js (hunk 1 above)
Comment (PaulSoonMinChoi) [reply app/src/components/Preferences/PreferencesCard.js:31]: Gap creates the space between the flex items.

**Comment by PaulSoonMinChoi** [comment]
Here are some of the screen shots! I will also make sure to make a description of the PR in the future! 

Screen Shot 2021-04-10 at 12 57 58 PM
Screen Shot 2021-04-10 at 12 57 42 PM

File: app/src/components/Preferences/PreferencesCardStyles.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles({
+  root: {
+    minWidth: 275,
+    borderTop: '15px solid #FF4F9E',
+    margin: '1rem',
+    padding: '1rem',
```
Comment (jonathanbutler7) [inline app/src/components/Preferences/PreferencesCardStyles.js:8]: Change this to `padding: '0 1rem',`

File: app/src/components/Preferences/PreferencesCard.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    <Card className={classes.root}>
+      <CardContent>
+        <div
+          style={{
+            display: 'flex',
+            alignItems: 'center',
+            justifyContent: 'space-between',
+          }}>
+          <Typography variant='h4'>Training</Typography>
+          <Typography variant='h6' className={classes.subBody}>
```
Comment (jonathanbutler7) [inline app/src/components/Preferences/PreferencesCard.js:22]: If you add a prop to Typography of `color="primary"` then you don't need this `classes.subBody` className and you use the colors in the theme object.

File: app/src/components/shared/Header.js [hunk 4]
```diff
@@ -1,15 +1,15 @@
 import React from 'react';
```
Comment (jonathanbutler7) [inline app/src/components/shared/Header.js:1]: Thanks for putting this in a `shared` folder

File: app/src/components/Preferences/PreferencesCardStyles.js [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+  },
+  body: {
+    margin: '0',
+    color: '#249BE5',
+  },
+  subBody: {
+    color: '#249BE5',
+  },
+  icon: {
```
Comment (jonathanbutler7) [inline app/src/components/Preferences/PreferencesCardStyles.js:17]: I think this `icon` class is unnecessary. I tried commenting it out and didn't see any of the styles change. Icon size should be defined in theme.js. Lmk if I'm missing something though!

File: app/src/components/Preferences/ReviewPreferences.js [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+        subtitle='Review the information is correct before creating'
+      />
+      <PreferencesCard />
+      <div
+        style={{
+          position: 'absolute',
+          bottom: 0,
+          right: 0,
+        }}>
+        <Button size='large'>cancel</Button>
```
Comment (jonathanbutler7) [inline app/src/components/Preferences/ReviewPreferences.js:20]: These buttons feel a little cramped down in the corner based on the mockups in the Jira ticket. Probably just needs a little margin to give them some breathing room.

**Review by jonathanbutler7 — CHANGES_REQUESTED** [review]

File: app/src/components/Preferences/PreferencesCard.js (hunk 1 above)
Comment (jonathanbutler7) [reply app/src/components/Preferences/PreferencesCard.js:31]: Cool, never heard of it!

File: app/src/components/Preferences/PreferencesCardStyles.js (hunk 5 above)
Comment (PaulSoonMinChoi) [reply app/src/components/Preferences/PreferencesCardStyles.js:17]: Thank you for catching that! I had no use for that.

**Pull request merged**